### PR TITLE
troubleshoot-usb-dongle.md: clarify language

### PR DESCRIPTION
**Pull request by Lotterleben**
Make the language around `loopback-nousb` a bit more clear: what is this for and what do you need to remember.

This was prompted by a conversation with @BriocheBerlin , so her feedback would be super welcome :)

### CODE REVIEW

File: embedded-workshop-book/src/troubleshoot-usb-dongle.md [hunk 1]
```diff
@@ -12,18 +16,38 @@ $ change-channel 20
...
 ``` console
 $ serial-term
 now listening on channel 20
 now listening on channel 20
 ```
 
-If that's the case you are good to go, return to the ["Interference"] section.
+Return to the ["Interference"] section.
+
+🔎 `serial-term` shows you the log output that the Dongle is sending to your computer via the serial interface (not over the wireless network!). After you've ran `change-channel`, it tells you that it is now listening for network traffic on channel 20. This is helpful for debugging, but not mission-critical.
```
Comment (BriocheBerlin) [inline embedded-workshop-book/src/troubleshoot-usb-dongle.md:29]: Nice! I think this is the part I didn't understand at first!

File: embedded-workshop-book/src/troubleshoot-usb-dongle.md [hunk 2]
```diff
@@ -12,18 +16,38 @@ $ change-channel 20
...
+Put the device in bootloader mode again. Now, run
+```console
+$ dongle-flash loopback-nousb21  # you can pick 11, 16, 21 or 26
+```
+
+❗️ The number in the `loopback-nousb*` file name is the radio channel the Dongle will listen on. This means that when you program the Development Kit to send data to the Dongle, you need to ensure they are communicating on the same channel by setting
+
+```rust
+/* make sure to pass the channel number of the loopback-nousb* program you picked */
+radio.set_channel(Channel::_21);
```
Comment (BriocheBerlin) [inline embedded-workshop-book/src/troubleshoot-usb-dongle.md:48]: 👍

**Review by BriocheBerlin — COMMENTED** [review]
Thanks @Lotterleben, I think this will help prevent confusion!

You could add on radio-puzzle.md, section `Dongle Responses` something saying that the messages will be received, even if the Dongle's USB functionality is not working correctly. But maybe this is obsolete now because hopefully clear from the start? 🤔

**Comment by Lotterleben** [comment]
Good point, I've added some more text :)

File: embedded-workshop-book/src/radio-puzzle.md [hunk 3]
```diff
@@ -16,6 +16,8 @@ Like in the previous sections the Dongle will listen for radio packets -- this t
 
 ## Dongle Responses
 
+❗ The Dongle responds to the DK's requests wirelessly (i.e. by sending back radio packets) as well. You'll see the dongle responses printed by the DK. This means you don't have to worry if serial-term doesn't work on your machine.
+
```
Comment (BriocheBerlin) [inline embedded-workshop-book/src/radio-puzzle.md:20]: Perfect!

**Review by Mirabellensaft — APPROVED** [review]

**Pull request merged**